frontend/services/locations/localtionsService.ts

```diff
@@ -1,4 +1,9 @@
-import { Locations, LocationsParams, LocationsTypes } from 'types/locations';
+import { useMemo } from 'react';
```

Comment:
There's a typo in the name of the file: `locationsService.ts`. I would also simply rename is as `frontend/services/locations.ts`.

Reply:
I think it would be better if I create a refactor PR to standardise the files names and paths.